pytest_twisted.py

```diff
@@ -1,4 +1,13 @@
 import inspect
+import sys
+
+ASYNC_AWAIT = sys.version_info >= (3, 5)
+
+if ASYNC_AWAIT:
+    import asyncio
+else:
+    asyncio = None
+
```

Comment:
Is there some reason for this import? (It seems unused..)

Reply:
Maybe in some previous life the code used it to check if some feature should be available?